Formula/docker-machine-driver-xhyve.rb

```diff
@@ -38,8 +40,12 @@ def install
 
       if build.with? "qcow2"
         build_tags << " qcow2"
+        ENV["LIBEV_FILE"] = "#{Formula["libev"].lib}/libev.a"
```

Comment:
If you're using the static library you can do `depends_on "libev" => :build` above.